docs/test_the_installation.rst

```diff
@@ -29,18 +29,8 @@ Log in to Both Servers via TTY
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 All access to the SecureDrop servers should be performed over SSH from the
-*Admin Workstation*. To aid in troubleshooting, physical logins via TTY are
-supported, but require 2FA to be configured. See the :doc:`2FA setup guide
-<servers_2fa>` for information how to enable console logins.
-
-Test the 2FA functionality by connecting a keyboard and display to each server,
-then login with the Admin username. You will need:
-
-* sudo passphrase for the admin username
-* TOTP code from a 2FA app such as FreeOTP
-
-Confirm that logging in via TTY prompts for a 2FA code, and that the code
-generated by your smartphone app permits logging in to an interactive shell.
+*Admin Workstation*. To aid in troubleshooting, physical logins via TTY is
```

Comment:
Nit: singular/plural mix-up (logins/is). A less jargon-y way to put it might be: "To aid in troubleshooting, login via a physical keyboard attached to the server is also supported."